daemon/algod/api/server/router_test.go

```diff
@@ -166,6 +178,7 @@ func (s *TestSuite) TestGetAsset() {
 	assert.Equal(s.T(), callsBefore+1, s.calls)
 }
 func (s *TestSuite) TestGetTransactionByID() {
+	partitiontest.PartitionTest(s)
```

Comment:
These probably need to be `partitiontest.PartitionTest(s.T())`

Reply:
ok let me update them